docs/content/configuration/policy-resource.md

```diff
@@ -94,7 +94,7 @@ policies:
 
 ### RateLimit
 
-> **Feature Status**: Rate-Limiting is available as a preview feature[^1]: We might introduce some backward-incompatible changes to the resource definition. The feature is disabled by default. To enable it, set the [enable-preview-policies](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#cmdoption-enable-preview-policies) command-line argument of the Ingress Controller.
+> **Feature Status**: Rate-Limiting is in preview status until release 2.1.2.[^1]
```

Comment:
```suggestion
> **Feature Status**: Rate-Limiting was in preview status up to release 2.1.2.[^1]
```
I think this would be more clear but not 100% sure. @nginxinc/nginx-docs maybe can suggest?

Reply:
I would remove the past tense reference to what was.  It should **not** be noted with each and every previous case that required the parameter.  Because the statement refers to what was and not what is.

I would have a top level release note statement: '"preview status" has been removed and is no longer necessary to unlock any capabilities.'

Reply:
yes.
OIDC is the only one that should mention that the preview flag is necessary.
And a new PR should introduce new wording as it would be a deprecation statement of the parameter.  Specific to the new parameter for OIDC Policy enablement.

The preview language in the Policy (this) document should be removed from all Policies except OIDC in this PR.

Reply:
Agreed @brianehlert

Reply:
Documentation updated as suggested